plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -17,34 +17,73 @@
  * under the License.
  */
 import React, { useState } from 'react';
+import { ColumnInstance, Column } from 'react-table';
+import {
+  FaSort,
+  FaSortUp, // asc
```

Comment:
instead of adding a comment which makes this readable in one place, you could `import { FaSortUp as FaSortAsc } ..` which will make it readable throughout?